### PR TITLE
Integration API release 2 examples

### PR DESCRIPTION
**Pull request by lyyder**
Adds a few examples the take the features of the Integration API release 2 in to use:
- upload an image
- approve user's listings
- update user metadata

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,3 +138,39 @@ If you have any questions, feel free to contact us by responding to this email.
...
+
+### Approve listings of a user
+
+Approves all listings of a given user that are in the `pendingApproval` state.
+The user is defined by passing an email address as the first parameter.
+
+```
+> node scripts/approve-listings-by-user.js john.doe@example.com                                                                               <<<
+Approved listing: Sauna by a lake (9f3419cc-49aa-4877-ac5b-7b25248414ef)
+Approved listing: Electric ciy sayna (9009efe1-25ec-4ed5-9413-e80c584ff6bf)
```
Comment (OtterleyW) [inline README.md]:
```suggestion
Approved listing: Electric city sauna (9009efe1-25ec-4ed5-9413-e80c584ff6bf)
```

File: scripts/update-user-metadata.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  clientId: process.env.FLEX_INTEGRATION_CLIENT_ID,
+  clientSecret: process.env.FLEX_INTEGRATION_CLIENT_SECRET,
+
+  // Normally you can just skip setting the base URL and just use the
+  // default that the `createInstance` uses. We explicitly set it here
+  // for local testing and development.
+  baseUrl: process.env.FLEX_INTEGRATION_BASE_URL || 'https://flex-api.sharetribe.com',
+});
+
+// Fetch user ID, fetch user's all listings, and approve them one by one
```
Comment (OtterleyW) [inline scripts/update-user-metadata.js]: Isn't this about updating the user metadata and not approving listings?

File: scripts/approve-listings-by-user.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+// Query params for approve listing request
+const listingsApproveParams = {
+  expand: true,
+  'fields.listing': 'title',
+};
+
+// query params for users/show request
+const usersShowParams = emailAddress => ({
+  email: emailAddress,
+  'fields.listing': 'none',
```
Comment (OtterleyW) [inline scripts/approve-listings-by-user.js]: Is it necessary to pass the `'fields.listing': 'none'` as param? In user metadata example we don't do that so what's the difference?

Comment (lyyder) [reply scripts/approve-listings-by-user.js]: It is supposed to narrow down the returned attributes, but now I see that it's targeting `listing` fields and our `show` endpoints don't even support the sparse attributes so I'll remove it. Probably just left there from some testing.

Comment (lyyder) [reply scripts/approve-listings-by-user.js]: Also, this was not even used 🤦‍♂.

**Pull request merged**